docs/content/latest/yugabyte-cloud/cloud-quickstart/cloud-build-apps/cloud-ysql-go.md

```diff
@@ -0,0 +1,179 @@
+    - **sslMode** - the SSL mode to use. Yugabyte Cloud [requires SSL connections](../../../cloud-secure-clusters/cloud-authentication/#ssl-modes-in-ysql); use `verify-full`.
+    - **sslRootCert** - the full path to the Yugabyte Cloud cluster CA certificate.
+
+3. Save the file.
+
+## Build and run the application
+
+First, initialize the `GO111MODULE` variable.
+
+```sh
+$ export GO111MODULE=auto
+```
+
+Import the Go PostgreSQL driver.
+
+```sh
+$ go get github.com/lib/pq
+```
+
+To start the application, run the following command.
+
+```sh
+go run sample-app.go
+```
+
+You should then see output similar to the following:
+
+```output
+>>>> Successfully connected to YugabyteDB!
+>>>> Successfully created table DemoAccount.
+>>>> Selecting accounts:
+name = Jessica, age = 28, country = USA, balance = 10000
+name = John, age = 28, country = Canada, balance = 9000
+>>>> Transferred 800 between accounts.
+>>>> Selecting accounts:
+name = Jessica, age = 28, country = USA, balance = 9200
+name = John, age = 28, country = Canada, balance = 9800
+```
+
+You have successfully executed a basic Go application that works with Yugabyte Cloud.
+
+## Explore the application logic
+
+Open the `sample-app.go` file in the `yugabyte-simple-go-app` folder to review the methods.
+
+### main
+
+The `main` method establishes a connection with your cluster via the Go PostgreSQL driver.
+
+```go
+psqlInfo := fmt.Sprintf("host=%s port=%d user=%s password=%s dbname=%s",
+    host, port, dbUser, dbPassword, dbName)
+
+if sslMode != "" {
+    psqlInfo += fmt.Sprintf(" sslmode=%s", sslMode)
+
+    if sslRootCert != "" {
+        psqlInfo += fmt.Sprintf(" sslrootcert=%s", sslRootCert)
+    }
+}
+
+db, err := sql.Open("postgres", psqlInfo)
+```
+
+### createDatabase
+
+The `createDatabase` method uses PostgreSQL-compliant DDL commands to create a sample database.
+
+```go
+Statement stmt = conn.createStatement();
+
+stmt = `CREATE TABLE DemoAccount (
+                    id int PRIMARY KEY,
+                    name varchar,
+                    age int,
+                    country varchar,
+                    balance int)`
+
+stmt = `INSERT INTO DemoAccount VALUES
+            (1, 'Jessica', 28, 'USA', 10000),
+            (2, 'John', 28, 'Canada', 9000)`
+```
+
+### selectAccounts
+
+The `selectAccounts` method queries your distributed data using the SQL `SELECT` statement.
+
+```go
+rows, err := db.Query("SELECT name, age, country, balance FROM DemoAccount")
+checkIfError(err)
+
+defer rows.Close()
+
+var name, country string
+var age, balance int
+
+for rows.Next() {
+    err = rows.Scan(&name, &age, &country, &balance)
+    checkIfError(err)
+
+    fmt.Printf("name = %s, age = %v, country = %s, balance = %v\n",
+        name, age, country, balance)
```

Comment:
You're missing `}` in the end - you need to close the `for rows.Next() {...}` block.